courses.json

```diff
@@ -1 +1 @@
\ No newline at end of file
+[]
```

Comment:
Hey @debajyotidasgupta I can see that the courses file is empty. We use this file for displaying the slots, courses, etc.

Can you please check once more if you used the right credentials?